tests/pre_merge/deploy/test_inferencer.py

```diff
@@ -49,7 +49,7 @@ def get_model_config(
 class TestInferencers:
     @pytest.mark.parametrize(
         "model_name",
-        ["padim", "stfpm", "patchcore", "dfm", "dfkde", "ganomaly", "cflow"],
+        ["cflow", "dfm", "dfkde", "ganomaly", "padim", "patchcore", "stfpm"],
```

Comment:
Should reverse distillation be added here?

Reply:
Yes, I still need to test it.